operator/src/monitoring/otel-config.yaml

```diff
@@ -0,0 +1,82 @@
+---
+receivers:
+  # Push based metrics
+  otlp:
```

Comment:
Which services use the push-based metrics vs which services set up the metrics that we are scraping using prometheus? Can these metrics be colliding? Can we add a comment or doc in terms of how we should name/label metrics to avoid such collisions

Reply:
Scraping will be performed on any pods in the namespace that have a port name "metrics"
https://github.com/3box/keramik/pull/198/files#diff-343efbc6f501d7065cfa0c42c6d66ea7b49ab1783578c7b695b6acf4c1728400R11-R45

Push based metrics would be anyone sending metric to the collector.
In keramik, I believe, and would like, that to be just the simulation manager and workers.
Since they're ephemeral jobs, they should push their metrics before the go away/stop.

Reply:
I think if we keep this separation, services pull, simulations push, we should avoid duplications.